go.mod

```diff
@@ -3,9 +3,9 @@ module decred.org/dcrctl
 go 1.13
 
 require (
-	decred.org/dcrwallet v1.2.3-0.20200504180520-7feccf86bd7d
+	decred.org/dcrwallet v1.2.3-0.20200519180100-f1aa4c354e05
 	github.com/decred/dcrd/dcrjson/v3 v3.0.1
-	github.com/decred/dcrd/dcrutil/v3 v3.0.0-20200421213827-b60c60ffe98b
+	github.com/decred/dcrd/dcrutil/v3 v3.0.0-20200520040632-c4751ba7284b
```

Comment:
Should be fine, but not strictly necessary.

Reply:
Figured I would just get this done while I was in the area. No harm right?